src/doc/yaml/abis.yaml

```diff
@@ -2101,21 +2101,16 @@ components:
           type: string
     TemplateFormat:
       type: string
-      enum:
-        - ISO_19794_2
-        - ISO_19794_2_NS
-        - ISO_19794_2_CS
-        - ISO_19794_2_2011
-        - ANSI_378_2009
-        - ANSI_378
-      description: Format of the template
+      description: |
+        Format of the template.
+        One of ISO_19794_2, ISO_19794_2_NS, ISO_19794_2_CS, ISO_19794_2_2011, ANSI_378_2009 or ANSI_378.
+        Can be extended to include additional proprietary template format
     QualityFormat:
       type: string
-      enum:
-        - ISO_19794
-        - NFIQ
-        - NFIQ2
-      description: Format of the quality
+      description: |
+        Format of the quality.
+        One of ISO_19794, NFIQ, or NFIQ2.
+        Can be extended to include additional proprietary quality format
```

Comment:
As per Template format. Enum seems safer to me, just with addition of CUSTOM as enum value. Just my opinion

Reply:
With CUSTOM as the value, we will need to provide the actual value also. So where/how do we do that? A free form text would be better I feel.